wye/profiles/urls.py

```diff
@@ -4,8 +4,8 @@
 urlpatterns = [
     url(r'^dashboard$', views.UserDashboard.as_view(),
         name="dashboard"),
-    url(r'^(?P<slug>[a-zA-Z0-9]+)/$',
+    url(r'^(?P<slug>[a-zA-Z0-9@.]+)/$',
         views.ProfileView.as_view(), name='profile-page'),
-    url(r'^(?P<slug>[a-zA-Z0-9]+)/edit$',
+    url(r'^(?P<slug>[a-zA-Z0-9@.]+)/edit$',
```

Comment:
Ah. didn't notice that subject of PR is not what the code does. The code is fine given the conversation.